src/util/graphic.js

```diff
@@ -75,6 +77,10 @@ export function extendShape(opts) {
     return Path.extend(opts);
 }
 
+export function fixClipWithShadow(conditionCheck) {
+    return fixPathClipWithShadow(Path.prototype.brush, conditionCheck);
```

Comment:
Is there a PR for the modification of `fixPathClipWithShadow` (about adding `conditionCheck`) ?
Does `sausage` will be used as a clipping path?

Reply:
Sorry, I forgot to make a PR. It's available now: https://github.com/ecomfe/zrender/pull/513/files